docs/user-guide/cli-using-usingcli.md

```diff
@@ -1,7 +1,35 @@
 # Using Zowe CLI
 
-Learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
+In this section, learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
 
 You can use the CLI interactively from a command window on any computer on which it is installed, or run it in a container or automation environment.
 
-**Tip:** If you want to use the CLI together with a screen reader to provide accessibility, we recommend using the Mac™ Terminal application enabled for Accessibility through [System Preferences > Accessibility](https://support.apple.com/zh-sg/guide/terminal/trml1020/mac). On Windows™, adjust the Properties settings in Command Prompt. For other operating systems, or for alternative terminals, check the specification for the terminal to ensure that it meets accessibility requirements.
\ No newline at end of file
+**Tip:** If you want to use the CLI together with a screen reader to provide accessibility, we recommend using the Mac™ Terminal application enabled for Accessibility through [System Preferences > Accessibility](https://support.apple.com/zh-sg/guide/terminal/trml1020/mac). On Windows™, adjust the Properties settings in Command Prompt. For other operating systems, or for alternative terminals, check the specification for the terminal to ensure that it meets accessibility requirements.
+
+## Supported CPU architecture, operating systems and package/resource managers
+
+Zowe CLI supports the following CPU architecture:
+- x64
+- Apple Silicon (M1+) with Rosetta
+    - The [IBM Db2 Database Plug-in for Zowe CLI](../user-guide/cli-db2plugin) has limited support on Apple Silicon. To use the Db2 plug-in, a complete re-install of Zowe CLI and CLI plug-ins is required. See [M1 processor installation](../user-guide/cli-db2-install-m1) for information.
+### Operating systems
+
+- MacOS 10.15+
+- Unix-like:
+   - [CentOS](https://www.centos.org/) 8+
+   - [Debian](https://www.debian.org/) 11+
+   - [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) 8+
+   - [Ubuntu](https://ubuntu.com/) 20.04+
+
+- Windows 10+ 
+
+### Package/resource managers
+
+- [NodeJS](https://nodejs.org/en)
+- [npm](https://www.npmjs.com/) 6+
+- [PnPM](https://pnpm.io/)
+- [Yarn](https://yarnpkg.com/) 
+
+<br/> 
+
+>Using Zowe CLI on [z/OS Unix Systems Services](https://www.ibm.com/docs/en/zos/2.4.0?topic=descriptions-zos-unix-system-services) is not supported at this time. If you would like to use USS in the future, show your interest by voting for the enhancement in the [Zowe CLI GitHub repository](https://github.com/zowe/zowe-cli/issues/1680). If enough votes are cast, the enhancement is added to the Zowe CLI dev team backlog.
```

Comment:
@adam-wolfe Let me know if this is good as a reference/link to the USS github issue

Reply:
I think we could remove the last sentence. We don't really have a threshold for how many 👍 we are looking for - it would just be another way to gauge interest so we can better prioritize the work.

Reply:
I think we do have a count -- at least according to the bot: 
<img width="443" alt="image" src="https://user-images.githubusercontent.com/109533923/230471796-c01916fe-f3af-4dd8-b679-33bc5c55755e.png">

but I'm removing the sentence regardless, thank u